packages/core/src/renderer/components/dock/create-resource/stop-watching-user-templates-on-quit.injectable.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { beforeQuitOfBackEndInjectionToken } from "../../../../main/start-main-application/runnable-tokens/before-quit-of-back-end-injection-token";
+import userTemplatesInjectable from "./user-templates.injectable";
+import loggerInjectable from "../../../../common/logger.injectable";
+
+const stopWatchingUserTemplatesOnQuitInjectable = getInjectable({
```

Comment:
Something wrong here? This runnable does not seem to get called (the logger.info() message never appears)

@jansav